src/Files.App/ViewModels/UserControls/SidebarViewModel.cs

```diff
@@ -399,15 +399,20 @@ private async Task AddElementToSection(INavigationControlItem elem, LocationItem
 				}
 			}
 
-			section.IsExpanded = Ioc.Default.GetRequiredService<SettingsViewModel>().Get(section.Text == "SidebarFavorites".GetLocalizedResource(), $"section:{section.Text.Replace('\\', '_')}");
+			var key = $"section:{section.Text.Replace('\\', '_')}";
+
+			section.IsExpanded = ApplicationData.Current.LocalSettings.Values.Get(key, section.Text == "SidebarFavorites".GetLocalizedResource());
```

Comment:
It's out of scope for this PR, but there is some room for further improvements in this area. Ideally this would be saved to `settings.json` like everything eles.